.github/workflows/ci.yml

```diff
@@ -77,3 +77,51 @@ jobs:
           git status --porcelain
           git diff --no-ext-diff --quiet --exit-code
         name: Check not out of date generated files
+
+  build-dev:
+    name: Build And Test (-dev.x)
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install -g pnpm@7.27.1
+        name: Install PNPM
+
+      # Install deps and build the bump package first
+      # - run: pnpm install
+      #   name: Install dependencies
+
+      # - run: npm run build
+      #   name: Build
+      #   working-directory: ./typespec-bump-deps
+
+      - run: npx -y @azure-tools/typespec-bump-deps@latest ./packages/cadl-ranch-specs/package.json ./packages/cadl-ranch/package.json ./packages/cadl-ranch-expect/package.json
```

Comment:
Is this actually what pulls the latest from typespec-azure?

Reply:
this will bump all the version in those pacakge.json to be the latest `-dev` available. This is the same package(defined in this repo) used by the emitter for their nightly build